package.json

```diff
@@ -1,18 +1,16 @@
 {
```

Comment:
looks like it does, the pack-translator *will* overwrite this file. sounds like we'll need to update pack-translator

---
In reply to: [85858482](https://github.com/openT2T/translators/pull/116#discussion_r85858482) [](ancestors = 85858482)